sources/packages/web/src/components/common/applicationTracker/Completed.vue

```diff
@@ -309,10 +330,13 @@ export default defineComponent({
       COEStatus,
       AssessmentTriggerType,
       StudentAppealStatus,
+      ApplicationEditStatus,
```

Comment:
`ApplicationEditStatus`  `ApplicationEditStatus` not required in the return.

Reply:
`ApplicationEditStatus` removed.